packages/venia-concept/_buildpack/create.js

```diff
@@ -105,7 +104,8 @@ function createProjectFromVenia({ fs, tasks, options }) {
                 delete config.projects.venia;
                 fs.outputJsonSync(targetPath, config, { spaces: 2 });
             },
-            '{CHANGELOG*,LICENSE*,_buildpack/*}': tasks.IGNORE,
```

Comment:
The changes remove the top-level `_buildpack` and any `__tests__` directories as well as their entire contents. Previously we were just contents and not directories which is why sub directories were being included.